tests/unit/flow/test_flow_yaml_parser.py

```diff
@@ -118,9 +118,14 @@ class DummyEncoder(Executor):
 
 def test_flow_yaml_override_with_protocol():
     from jina.enums import GatewayProtocolType
+
     f1 = Flow.load_config('yaml/examples/faiss/flow-index.yml')
     assert f1.protocol == GatewayProtocolType.GRPC
-    f2 = Flow.load_config('yaml/examples/faiss/flow-index.yml', override_with={'protocol': 'http'})
+    f2 = Flow.load_config(
+        'yaml/examples/faiss/flow-index.yml', override_with={'protocol': 'http'}
```

Comment:
please make sure the test path is relative to the current path of the test. Check the usage of `cur_dir` in other tests

Reply:
Addressed, please see [7cbd79a](https://github.com/jina-ai/jina/pull/2901/commits/7cbd79a9116e467c747f48c201522686631314e8) for the change.